docs/howto/operator.rst

```diff
@@ -284,6 +284,148 @@ If the source code for your function is in Google Source Repository, make sure t
 your service account has the Source Repository Viewer role so that the source code
 can be downloaded if necessary.
 
+CloudSqlInstanceDatabaseCreateOperator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Creates a new database inside a Cloud SQL instance.
+
+For parameter definition take a look at
+:class:`~airflow.contrib.operators.gcp_sql_operator.CloudSqlInstanceDatabaseCreateOperator`.
+
+Arguments
+"""""""""
+
+Some arguments in the example DAG are taken from Airflow variables:
```

Comment:
environment variables

Reply:
Thanks! Corrected this for all 6 Cloud SQL operators in the `operator.rst`. Also I noticed that the example DAG's docstring was out of date, so I updated it too.